content/blog/2020-08-14-spreadsheet-workflows-using-r/index.Rmarkdown

```diff
@@ -0,0 +1,242 @@
+---
+title: Spreadsheet Workflows in R
+date: '2020-08-14'
+slug: spreadsheets-using-r
+categories:
+  - learn
+tags:
+  - spreadsheets
+  - tidyverse
+author:
+  - Devin Johnson
+photo:
+  url: https://unsplash.com/photos/Wpnoqo2plFA
+  author: Mika Baumeister
+---
+
+There's so much to discover and learn when working with spreadsheets. As a summer intern at RStudio my project has focused on creating a resource that lies at the intersection between spreadsheets and R, aimed at users who either exclusively use one or the other or whose work lives in the intersection of the two. This project has focused on a plethora of topics from comparing numerous R package functionalities that work with spreadsheets, to writing about tips and tricks on best practices for working in both a spreadsheet software and in RStudio. 
+
+In this post I'd like to share a part of this work, specifically focused on the R versions of common spreadsheet workflows. To do that we'll assume the role of a data analyst in two hypothetical scenarios and walk through the tidyverse equivalent of common analyses and tasks one is likely to come across when working with data stored in spreadsheets.   
+
+For our first story, it's day one as a new data analyst and our boss has asked us to take a look at the company's finances.
+They send you a spreadsheet, available in an Excel workbook, and ask you to analyze spending patterns over the current year and give your assessment into where spending is going overboard.
+
+We've decided to run our analysis in RStudio to take advantage of data wrangling and visualization functions from the tidyverse as well as to create a reproducible workflow in case we want to revisit this analysis later.  First, let's load in our data and take a look at it in RStudio. 
```

Comment:
```suggestion
We've decided to run our analysis in R to take advantage of data wrangling and visualization functions from the tidyverse as well as to create a reproducible workflow in case we want to revisit this analysis later.  First, let's load in our data and take a look at it in R. 
```